packages/components/src/components/Card/Card.docs.mdx

```diff
@@ -1,86 +1,288 @@
----
-title: Card
----
-
-import { Preview, Props } from "@lighting-beetle/lighter-styleguide";
-import Card, { CardContent, CardHeader, CardFooter } from ".";
-import Button from "./../Button";
-
-# Card
-
-A card serves as an entry point to more detailed information. Cards may contain rich content, such as photos, text, link, list, tables, you name it.
-
-<Card>
-  <CardContent>
-    <h2>Card</h2>
-    <p>Content</p>
-  </CardContent>
-</Card>
-
-## Variants
-
-### Basic
-
-<Preview>
-  <Card>
-    <CardContent>
-      <h2>Card</h2>
-      <p>Content</p>
-    </CardContent>
-  </Card>
-</Preview>
-
-### Card header and footer
-
-Card header holds a title and possibly an action button. Card footer is usually used for additional information or confirm/cancellation buttons.
-
-<Preview>
-  <Card>
-    <CardHeader title="Title" />
-    <CardContent>Content</CardContent>
-    <CardFooter>
-      <Button type="primary">CTA</Button>
-    </CardFooter>
-  </Card>
-</Preview>
-
-### Card header actions
-
-To add an action button in card header, a Bar is added to wrap things nicely.
-
-<Preview>
-  <Card>
-    <CardHeader title="Title" actions={<Button>Action</Button>} />
-    <CardContent>Content</CardContent>
-  </Card>
-</Preview>
-
-### Multiple content sections
-
-Multiple contents helps with visual separation of content.
-
-<Preview>
-  <Card>
-    <CardContent>
-      <h2>Card</h2>
-      <p>Content</p>
-    </CardContent>
-    <CardContent>
-      <p>Second content</p>
-    </CardContent>
-  </Card>
-</Preview>
-
-## Props
-
-<Props component={Card} />
-
-<Props component={CardContent} />
-
-<Props component={CardHeader} />
-
-<Props component={CardFooter} />
-
-## Accessibility
-
-The `<Card />` component is purely presentational and does not carry any semantic meaning. The only thing you should care about is the usage of correct heading level in `<CardHeader title="" />`
+---
+title: Card
+---
+
+import Link from "next/link";
+import { Preview, Props } from "@lighting-beetle/lighter-styleguide";
+
+import Icon from "../Icon";
+import Button from "../Button";
+
+
+import Card, { CardSection, CardImage, CardTitle, CardAction } from ".";
+
+
+Navigačný komponent. Môže obsahovať obrázok, nadpis, text aj button/link.
+
+Karta slúži ako vstupný bod k podrobnejším informáciám. Karty môžu obsahovať bohatý obsah, ako sú fotografie, text, odkaz, zoznam, tabuľky,..
+
+### Ukážka
+
+<Preview>
+  <Card>
+    <CardSection>
+      <CardTitle>Simple card</CardTitle>
+      <p>Card Content</p>
+    </CardSection>
+  </Card>
+</Preview>
+
+### Použitie
+
+<p>
+  Pri klikateľnej kartičke je potrebné manuálne poslať do <code>Card</code>
+  komponentu <code>isClickable</code> prop, vďaka ktorému kartička nadobudne štýly,
+  ktoré signalizujú, že je klikateľná.

```

Comment:
toto vyrendruje div, ktory na hover vyzera ako klikatelny, ale klikanie si musis dorobit. bolo by velmi vhodne pouzit `anchor` alebo `button` element podla usecase, spolu s takymto niecim https://ods1.netlify.app/components/block-action

Reply:
Pouzivame na to CardAction, co je popisane tu nizsie o 2 paragrafy. To nie je v poriadku? Mal som pocit, ze toto cele vzislo na Dovere po tvojej konzultacii s Davidom, nie je to tak?

Reply:
Resp. inak sa spytam - je podla teba ten pattern, co je v dokumentacii v casti klikatelna karticka pristupny? Odhliadnuc od API, ktore tu pouzivame na vytovrenie klikatelnej karticky.